libraries/AP_Motors/AP_MotorsTri.cpp

```diff
@@ -17,6 +17,7 @@
 #include <AP_Math/AP_Math.h>
 #include <GCS_MAVLink/GCS.h>
 #include "AP_MotorsTri.h"
+#include <AP_Vehicle/AP_Vehicle.h>
```

Comment:
This include is required so that the APM_BUILD_TYPE macro works?  I would have thought this came from something lower down.  If possible I like to keep the includes minimalist so we avoid include spaghetti.

Reply:
Your correct, this include was not required, I have removed